src/test/java/com/kylenicholls/stash/parameterizedbuilds/eventHandlers/PushHandlerTest.java

```diff
@@ -69,7 +69,7 @@ public void setup() {
 
         when(minimalRef.getId()).thenReturn(BRANCH_REF);
         jobBuilder = new Job.JobBuilder(1).jobName("").buildParameters("").branchRegex("")
-                .pathRegex("");
+                .pathRegex("").ignoreCommitMsg("").ignoreComitters("");
```

Comment:
Could you add a test or two in this file that ensures the ignore fields work with actual values?

Reply:
> Could you add a test or two in this file that ensures the ignore fields work with actual values?

Hi @a-hacker , Will need bit of your help 
I am not much familiar with the mocking so when i am trying to write tests `commitService.getCommit(commitRequest).getMessage();` always throws NullPointerException  how can i mock so it will return the predefined message ?

Reply:
Sorry for the delay, I've been out on vacation.

You'll need to add a few lines to the setup method to mock out the commit. You can create the mock using `Commit commit = mock(Commit.class);`. This creates a dummy object and to have it returned from the commitService, add a line like `when(commitService.getCommit(any())).thenReturn(commit);` to the setup too. Finally, you can do something similar to return a specific commit message: `when(commit.getMessage()).thenReturn("my_commit_message");`.

Reply:
Thank you very much, it worked.
I have added few test cases for both scenarios, So with this i have resolved all the comments. Please let me know if anything else need to be modified before merge